src/components/Logo/index.js

```diff
@@ -54,7 +54,7 @@ export const Logo = ({ tall, ...props }) => (
       data-testid="logo"
       alt={configuration.language.logoDescription}
       tall={tall}
-      src={configuration.logo.url}
+      src="/assets/images/logo.svg"
```

Comment:
This way only svg logos can be used, not png logos, right?

Reply:
Yeah, I realized that. I think that the logo URL is something we cannot change that easily. If at all.